fetools-app/src/index.css

```
@@ -94,6 +94,12 @@
 @layer base {
 
   @layer InputComponents{
+
+    *:focus-visible{
+      margin: 2px;
+      outline: 2px solid #4A256F;
```

We don't want to change margin on focus, or things will jump around on screen. The focus color should be #7F40BF.

```suggestion
      outline: 2px solid #7F40BF;
```